TROUBLESHOOTING.md

```diff
@@ -0,0 +1,42 @@
+# Troubelshooting the Devfile Registry
+
+## Collecting Logs
+
+Logs can be collected for both the devfile index server, and the OCI registry server in the devfile registry, that should assist with debugging any potential issues.
+
+To retrieve the logs from the devfile index server:
+
+```bash
+kubectl logs <devfile-registry-pod> -c devfile-registry
```

Comment:
The container name is `devfile-registry-bootstrap` or `devfile-registry`? I am using operator to deploy the registry to my minikube cluster, the container name is `devfile-registry-bootstrap`.
```
kubectl logs devfile-registry-6675b84db4-gr5xf                                                                      1 ✘  minikube/test ⎈  14:18:40 
error: a container name must be specified for pod devfile-registry-6675b84db4-gr5xf, choose one of: [devfile-registry-bootstrap oci-registry]
```

Reply:
D'oh, you're right, I was looking at the hosted registry, which uses something different for _some_ reason.